src/drivers/net/ena.c

```diff
@@ -206,23 +207,39 @@ static int ena_create_admin ( struct ena_nic *ena ) {
 	}
 	memset ( ena->aq.req, 0, aq_len );
 
+	/* Allocate AENQ queue */
```

Comment:
"Allocate admin event queue" for consistency with the other comments?

Reply:
I'll put in "async event queue". This is not an admin queue :)